theme/management/commands/update_used_storage.py

```diff
@@ -13,6 +15,21 @@ class Command(BaseCommand):
            "brought in sync with iRODS quota AVUs"
 
     def handle(self, *args, **options):
-        for u in User.objects.all():
+        errors = {}
+        users = User.objects.all()
+        count = users.count()
+        for i, u in enumerate(users, start=0):
+            i += 1
             if UserQuota.objects.filter(user=u).exists() and u.is_active and not u.is_superuser:
```

Comment:
Instead of checking `u.is_active` and` u.is_superuser` here we can filter out these users in line 19.